### PR TITLE
Nmc/1906 Text Editor

### PR DESCRIPTION
**Pull request by surinder-tsys**
- [ ] Tests written, or not not needed

### CODE REVIEW

File: app/src/main/java/com/owncloud/android/ui/activity/TextEditorWebView.kt [hunk 1]
```diff
@@ -75,9 +75,9 @@ class TextEditorWebView : EditorWebView() {
             WebSettingsCompat.setForceDark(getWebView().settings, WebSettingsCompat.FORCE_DARK_ON)
         }
 
-        getWebView().setDownloadListener { url, _, _, _, _ -> downloadFile(Uri.parse(url)) }
+        webView.setDownloadListener { url, _, _, _, _ -> downloadFile(Uri.parse(url)) }
 
-        loadUrl(intent.getStringExtra(ExternalSiteWebView.EXTRA_URL))
+        loadUrl(null)
```
Comment (tobiasKaminsky) [inline app/src/main/java/com/owncloud/android/ui/activity/TextEditorWebView.kt:80]: Why is this needed?
Previously it was possible to open an url that was passed to to this web view.

Comment (surinder-tsys) [reply app/src/main/java/com/owncloud/android/ui/activity/TextEditorWebView.kt:80]: @tobiasKaminsky This is the fix we received from your end for Endless loading when switching view modes.
Here is the PR: https://github.com/nextcloud/android/pull/11402 and support link: https://support.nextcloud.com/#ticket/zoom/26303
